molviewspec/molviewspec/builder.py

```diff
@@ -224,6 +229,20 @@ class Parse(_Base):
     """
     Builder step with operations needed after parsing structure data.
     """
+    def raw_volume(
+        self,
+        *,
+        source: RawVolumeSourceT,
+        options: RawVolumeOptionsT | None = None,
+        additional_properties: AdditionalProperties = None,
+    ) -> RawVolume:
+        """
+        Create a raw volume
+        """
+        params = make_params(RawVolumeParams, locals(), type="raw_volume")
+        node = Node(kind="raw_volume", params=params, additional_properties=additional_properties)
+        self._add_child(node)
+        return RawVolume(node=node, root=self._root)
```

Comment:
I don't understand why we call it "raw volume". Wouldn't just "volume" be enough? Do we need to contrast it with some other type of volume which is less raw?
To me "raw volume" sounds a bit like the volumetric data are included directly in MVS, instead of being read from a referenced file.

This applies to multiple classes, methods, type names.